docs/configuration.asciidoc

```diff
@@ -1046,6 +1046,32 @@ you must instead set the `LogLevel` for the internal APM logger under the `Loggi
 | `Error`                 | String
 |============
 
+[float]
+[[config-trace-context-ignore-sampled-false]]
+==== `TraceContextIgnoreSampledFalse`
```

Comment:
Wondering if there might be a better name for this setting? It's a little awkward with the sampled flag value to which it applies included in the name, but it's hard coming up with something clear 😃 

I know it's more verbose, but maybe `IgnoreTraceContextSampledFalseNoAgentTracestate` better describes (and aligns with `IgnoreMessageQueues` naming)?

Reply:
That sounds super verbose to me. I think this is complicated enough that we won't find a simple name - I think the docs explains it well and the shorter name is easier.